README.md

```diff
@@ -1,2 +1,20 @@
 # taar
 Telemetry-Aware Addon Recommender
+
+## How does it work?
+The recommendation strategy is implemented through the [RecommendationManager](taar/recommenders/recommendation_manager.py). Once a recommendation is requested for a specific [client id](https://firefox-source-docs.mozilla.org/toolkit/components/telemetry/telemetry/data/common-ping.html), the recommender iterates through all the registered models (e.g. [CollaborativeRecommender](taar/recommenders/collaborative_recommender.py)), in the order they were registered in, and returns the results from the first model module that can perform a recommendation.
```

Comment:
nit: phrasing:
> in the order they were registered in

could read smoother like: linearly in their registered order. Results are returned from the first model...